tests/pytests/test.py

```diff
@@ -551,7 +551,7 @@ def testExplain(env):
         'ft.create', 'idx', 'ON', 'HASH',
         'schema', 'foo', 'text', 'bar', 'numeric', 'sortable',
         'v', 'VECTOR', 'HNSW', '6', 'TYPE', 'FLOAT32', 'DIM', '2','DISTANCE_METRIC', 'L2', 't', 'TEXT'))
-    q = '(hello world) "what what" hello|world @bar:[10 100]|@bar:[200 300]'
+    q = '(hello world) "what what" (hello|world) (@bar:[10 100]|@bar:[200 300])'
```

Comment:
Was also mentioned elsewhere, IIRC,
Suggest to keep the previous test case as well (without the parentheses)

Reply:
This [comment](https://github.com/RediSearch/RediSearch/pull/2596#discussion_r816340453) in #2596 is also relevant here

Reply:
OK. Though I think the `test_parser.py` tests covers these differences and this test can test for only a specific query